pkg/collector/corechecks/embed/logs-agent.go

```diff
@@ -68,11 +69,12 @@ func (c *LogsCheck) Run() error {
 func (c *LogsCheck) Configure(data check.ConfigData, initConfig check.ConfigData) error {
 	here, _ := osext.ExecutableFolder()
 	confd := config.Datadog.GetString("confd_path")
+	ddcondig := filepath.Clean(filepath.Join(confd, ".."))
```

Comment:
I think this is probably safer:
```
config.Datadog.ConfigFileUsed()
```
link: https://godoc.org/github.com/spf13/viper#ConfigFileUsed

In case `confd` is somehow overriden which I believe is supported with flags, environment variables. Other than that this is 👍

Reply:
+1

Reply:
the logs-agent currently relies on datadog.yaml and conf.d to be in the same directory. Looks like we should instead pass both the configDileUsed and the confd path.

Quick question: Do people always put config files in `config.Datadog.GetString("confd_path")` or do we look at multiple places?